### ISSUE TITLE
The accuracy of using Yolov5 for dust detection in ports is not high, and there will be a large number of false detections, especially at night

### ISSUE THREAD

**Issue by hdjhasd** [ultralytics/yolov5#13495]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

The accuracy of using Yolov5 for dust detection in ports is not high, and there will be a large number of false detections, especially at night

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hdjhasd, thank you for your interest in YOLOv5 🚀! Please visit our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) to get started, where you can find quickstart guides for common tasks like [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/) all the way to advanced optimization techniques like [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution/).

If this is a 🐛 Bug Report, please provide a **minimum reproducible example (MRE)** to help us investigate this further. This should include details such as custom dataset samples, relevant training or inference code snippets, and complete logs for better debugging.

If this is a ❓ Question related to your dust detection task, please make sure to share:
- Information about your custom dataset (e.g., size, quality, class balance)
- Examples of false detections, particularly at night, to better understand the issue
- Details of the model configuration or modifications you may have made
- Your training logs and hyperparameters used (learning rate, batch size, etc.)
- Steps you've already taken using our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)

## Requirements

[**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed, including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). To set up an environment:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

## Environments

YOLOv5 can be run in the following verified environments, all preconfigured with essential dependencies like [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with free GPUs: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are passing. These tests confirm YOLOv5's functionality for [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py), and [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) across various environments.

This is an automated response to assist you faster 😊. An Ultralytics engineer will take a look at your issue and provide additional support soon.

**Comment by pderrenger** [comment]
@hdjhasd thank you for your question. For improving accuracy and reducing false detections, especially in challenging scenarios like night-time dust detection, ensure your training dataset includes diverse and representative images of the deployment environment (e.g., day and night conditions). Verify that annotations are accurate and consistent, and consider adding background images to reduce false positives. You can also experiment with hyperparameter tuning or use Test-Time Augmentation (TTA) for better results. For more guidance, refer to the [Tips for Best Training Results](https://docs.ultralytics.com/yolov5/tutorials/tips_for_best_training_results/) documentation. Let us know if you need further clarification!
